xarrayutils/utils.py

```diff
@@ -887,3 +887,40 @@ def concat_dim_da(data, name):
     return xr.DataArray(data, dims=[name],
                         coords={name: (name, data)},
                         name=name)
+
+
+
+def da_detrend(a, b, start_shift=0, timedim='time'):
```

Comment:
I think we can name this just `detrend`, or do you think there is a more clear name?

Reply:
sure since it is built specifically fro xarray, probably no need to specify.
I would specify time maybe, but we are going to generalize that so 'detrend' should be a sufficient name once the switch is in